packages/font/src/font.tsx

```diff
@@ -74,5 +74,3 @@ export const Font: React.FC<Readonly<FontProps>> = ({
   `;
   return <style dangerouslySetInnerHTML={{ __html: style }} />;
 };
-
-Font.displayName = "Font";
```

Comment:
Should we remove the `displayName` for all other components as well?

Reply:
no, when the component uses `forwardRef` we should have the displayName, but in this case the component is not forwarding the ref